ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1884,3 +1884,155 @@ def _distributed(self) -> bool:
         if self.__distributed is None:
             self.__distributed = self.parameters.numcpu > 1
         return self.__distributed
+
+    @wraps(_MapdlCore.ndinqr)
+    def ndinqr(self, node, key, **kwargs):
+        """Wrap the ``ndinqr`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().ndinqr(node, key, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
+        return value
```

Comment:
No need to keep a temporary reference to a single use variable. Simply return it.
```suggestion
        return self.scalar_param(pname)
```